app/views/viewings/edit.html.erb

```diff
@@ -1,3 +1,7 @@
-<%= viewing.theatername %>
-<%= viewing.playtitle %>
-<%= viewing.date %>
\ No newline at end of file
+<h1>Edit Viewing</h1>
+
+<%= form_tag('/viewings/@viewing.id/update') do %>
```

Comment:
Ah. Yes. Concatenation! I was just experimenting with that as we speak! I will see what I can get fixed here!